### PR TITLE
feat: cleanup operator for wrong references

### PR DESCRIPTION
**Pull request by dejanzele**
## Pull request description 

Operator cleanup

## Checklist (choose whats happened)

Cleanup for wrong references

- [ ] breaking change! (describe)
- [x] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

- issues with webhooks due to invalid references

### CODE REVIEW

File: config/crd/patches/webhook_in_tests.yaml [hunk 1]
```diff
@@ -14,4 +14,4 @@ spec:
           path: /convert
       conversionReviewVersions:
       - v1
-      - v2
+      - v1beta1
```
Comment (vsukhin) [inline config/crd/patches/webhook_in_tests.yaml:17]: why did we remove v2?

File: config/webhook/manifests.yaml [hunk 2]
```diff
@@ -3,20 +3,21 @@ apiVersion: admissionregistration.k8s.io/v1
...
+  name: webhook-admission
 webhooks:
   - admissionReviewVersions:
       - v1
+      - v1beta1
     clientConfig:
       service:
         name: webhook-service
         namespace: system
-        path: /mutate-scripts-tests-testkube-io-v1-script
```
Comment (vsukhin) [inline config/webhook/manifests.yaml:14]: I beleive, we still have scripts crd, it's kind of legacy, might be we can remove it now

**Review by vsukhin — APPROVED** [review]
it's okay in general, a few small questions

File: config/crd/patches/webhook_in_tests.yaml (hunk 1 above)
Comment (dejanzele) [reply config/crd/patches/webhook_in_tests.yaml:17]: conversionReviewVersion is the version of the Kubernetes resource called ConversionReview which is created by the control plane when it wants to trigger a resource conversion in the operator.
The current supported versions of ConversionReview are v1 and v1beta1

File: config/manager/kustomization.yaml [hunk 3]
```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
-  newTag: latest
+  newName: kubeshop/testkube-operator
+  newTag: 1.5.9
```
[the remark this answers is not in the document]
Comment (dejanzele) [reply config/manager/kustomization.yaml:16]: I'll revert this change, but this technically isn't used anywhere, as we don't use the kustomize script for deployment, instead we use kustomize to generate the templates and move them in our Operator Helm chart manually

**Pull request merged**